packages/@react-native-windows/tester/src/js/examples/Image/ImageExample.windows.js

```diff
@@ -24,11 +24,8 @@ const {
 
 const IMAGE1 =
   'https://www.facebook.com/assets/fb_lite_messaging/E2EE-settings@3x.png';
-
 const IMAGE2 =
-  // [Windows #13284 - this image crashes e2etests]
-  // 'https://www.facebook.com/ar_effect/external_textures/648609739826677.png';
-  'https://www.facebook.com/ads/pics/successstories.png';
+  'https://www.facebook.com/ar_effect/external_textures/648609739826677.png';
```

Comment:
It may be best to add any desired images to the test web site so we eliminate external web dependencies.

Reply:
That certainly would be ideal. These are react-native core's tests.  For now I think we should continue to align with them.